Content.Shared/Humanoid/HumanoidVisualLayers.cs

```diff
@@ -1,35 +1,32 @@
-﻿using Content.Shared.Humanoid.Markings;
-using Robust.Shared.Serialization;
+﻿using Robust.Shared.Serialization;
 
-namespace Content.Shared.Humanoid
-{
-    [Serializable, NetSerializable]
-    public enum HumanoidVisualLayers : byte
-    {
-        Special, // for the cat ears
-        Tail,
-        Hair,
-        FacialHair,
-        UndergarmentTop,
-        UndergarmentBottom,
-        Chest,
-        Head,
-        Snout,
-        HeadSide, // side parts (i.e., frills)
-        HeadTop,  // top parts (i.e., ears)
-        Eyes,
-        RArm,
-        LArm,
-        RHand,
-        LHand,
-        RLeg,
-        LLeg,
-        RFoot,
-        LFoot,
-        Handcuffs,
-        StencilMask,
-        Ensnare,
-        Fire,
+namespace Content.Shared.Humanoid;
 
-    }
+[Serializable, NetSerializable]
+public enum HumanoidVisualLayers : byte
```

Comment:
This PR features quite a few changes that are hard to diff and statically analyze; you could consider reducing the amount of tidy-up that isn't essential if  this takes time to get reviewers.